### ISSUE TITLE
Increasing delay at night

### ISSUE THREAD

**Issue by OskarDamkjaer** [OskarDamkjaer/FirefoxDelayWebpage#15]
I find myself wanteing a longer delay when i should be sleeping

**Comment by herrbrixter** [comment]
I agree. I want a longer delay between 11 PM to 7 AM so I am forced to sleep.

**Comment by OskarDamkjaer** [comment]
Hey! Happy you found my repo! I've not done updates in a long time, as I started my first full time job right after launching the extension.. Hopefully I can look into this soon as in may I'll switch to 4 day work week 😄

**Comment by herrbrixter** [comment]
No worries. Take your time.

Having a separate delay for night mode would be appropriate. Let's say starting from 10 PM, the delay can't be less than 30 seconds. The minimum delay is to take into consideration #8 since the delay could have increased to more than 30 seconds by 10 PM.
